fm4tlp/tgbl_flight_negatives.py

```diff
@@ -26,14 +26,15 @@
 import os
 import pickle
 import random
+import sys
 
 from absl import app
 from absl import flags
 import numpy as np
 import pandas as pd
 import tensorflow.compat.v1 as tf
+from tensorflow.compat.v1.io import *
```

Comment:
Please remove this import (we cannot ever accept module-wide imports i.e. with *). tf is already accessible via `import tensorflow.compat.v1 as tf` above. Call gfile with `tf.io.gfile` throughout. Thanks